yautpa-desktop/src/components/Countdown/index.scss

```diff
@@ -1,20 +1,26 @@
 @mixin button() {
-    color: white;
-    background-color: black;
-    border: white 1px solid;
-    border-radius: 16%;
+    color: #0051ff;
+    background-color: #ffffff;
```

Comment:
This is literally just black in hex. Please use black instead, since it's easier to read

Reply:
nvm im stupid